vtysh/bgp_vty.c

```diff
@@ -4875,6 +4934,8 @@ cli_neighbor_timers_execute(char *vrf_name, int argc, const char *argv[])
     const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor;
```

Comment:
Not certainly in that way. The operation is successful because set values operating for group (if previously are set), but not different from them. So, values synchronized or not and in this sense there is no incorrect alternatives.